tests/Feature/CollectionTest.php

```diff
@@ -19,7 +19,10 @@
         dimension: 128,
     );
 
-    expect($response->json('code'))->toEqual(200);
+    // Status 0 and empty data is a success response
+    expect($response->status())->toEqual(200);
+    expect($response->json('code'))->toEqual(0);
```

Comment:
Note that code `0` is now a success response, per the documentation.
E.g. https://milvus.io/api-reference/restful/v2.5.x/v2/Collection%20(v2)/List.md